src/index.html

```diff
@@ -1,12 +1,55 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body
+    class="page__content"
+  >
+    <main
+      class="page__main"
+    >
+      <div
+        class="calendar calendar--start-day--sun calendar--month-length-31"
+      >
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
leave just calendar__day on each day

```suggestion
        <div class="calendar__day></div>
```